src/status_im/chat/styles/input/input.cljs

```diff
@@ -11,7 +11,7 @@
 (def color-command "#70777d")
 (def color-send "rgb(98, 143, 227)")
 
-(def max-input-height 66)
+(def max-input-height 152)
```

Comment:
Arbitrary number?

Reply:
As ticket states- there should fit more lines into chats input.

Reply:
Input already grows in height (you can see this behavior in current app). How does this work with different screen resolution?

Reply:
does it grow when input is not typed? like text is updated from subscription?

Reply:
Not sure but now we have this logic spread in 2 different places?
Also I would expect the width to be deduced from the text width so that it will always be right, no matter the platform.

Reply:
it now deduces from "Text" class element of given ~width~, because "basic-text-input" is already styled by "input-view"

Reply:
Still `152` is arbitrary right? Does it reflect something in particular?

Reply:
Yes it is. I just multiplied min-input-height by 4 - so reflects 4 lines?!?
but I am happy to change it if you think so.

Reply:
Ok makes sense. Could you make it explicit then? As is `(* 4 min-input-height)` ?

Reply:
sure. that makes sense :)